guides/v2.2/config-guide/multi-site/change-increment-id.md

```diff
@@ -0,0 +1,84 @@
+---
+group: configuration-guide
+title: Change increment ID for a database entity on particular store
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+This article discusses how to change the increment ID for a Magento database (DB) entity (order, invoice, credit memo, etc.) on a particular Magento store using the `ALTER TABLE` SQL statement.
+
+## Affected versions
+
+-  Magento Commerce (on premise): 2.x.x
+-  Magento Commerce (Cloud): 2.x.x
+-  MySQL: any supported version
+
+## When would you need to change increment ID
+
+You might need to change the increment ID for new DB entities in these cases:
+
+-  After a hard backup restore on a Live site
+-  Some order records have been lost, but their IDs are already being used by payment gateways (like PayPal) for your current Merchant account. Such being the case, the payment  gateways stop processing new orders that have the same IDs, returning the "Duplicate invoice id" error
+
+{: .bs-callout-info}
+You may also fix the payment gateway issue for PayPal by allowing multiple payments per invoice ID in PayPal's Payment Receiving Preferences. See the Knowledge Base article: [PayPal gateway rejected request - duplicate invoice issue][] .
+
+## Prerequisite steps
+
+1. Find stores and entities for which the new increment ID should be changed.
+1. Connect to your MySQL DB.
+   For Magento Commerce (Cloud), at first, you need to SSH to your environment.
```

Comment:
Variable for Magento Commerce cloud is `{{site.data.var.ece}}`

Reply:
Thanks